tests/test_websocket.py

```diff
@@ -310,42 +311,56 @@ class WebsocketWriterTests(unittest.TestCase):
 
     def setUp(self):
         self.transport = unittest.mock.Mock()
-        self.writer = websocket.WebSocketWriter(self.transport)
```

Comment:
No need to overwrite, just create an another one `writer = websocket.WebSocketWriter(...)` when it tends to be different.